source/scripts/Provider/EthereumProvider.ts

```diff
@@ -1,28 +1,49 @@
-export const EthereumProvider = () => {
-  const getNetwork = () => {};
-
-  const getAccounts = () => {};
-
-  const getChainId = () => {};
-
-  const getAddress = () => {};
+import { Web3Accounts } from '@syspollum/sysweb3-keyring';
+import {
+  getNftImage as web3NFTImage,
+  getTokenIconBySymbol as getTokenIcon,
+} from '@syspollum/sysweb3-utils';
+import { web3Provider as provider } from '@syspollum/sysweb3-network';
 
-  const getBalance = () => {};
-
-  const getXpub = () => {};
-
-  const getTokens = () => {};
-
-  const getState = () => {};
+export const EthereumProvider = () => {
```

Comment:
we need some more methods available to websites like

- get address
- get network
- get state
- get accounts
- get chain id for the current provider